spotify_player/src/event/mod.rs

```diff
@@ -220,6 +220,10 @@ pub fn handle_action_in_context(
                 }
                 ui.popup = None;
             }
+            Action::JumpToTrack => {
+                client_pub.send(ClientRequest::JumpToTrack(track.id))?;
+                ui.popup = None;
```

Comment:
no need to define additional `ClientRequest`, you can handle the action directly here. The `jump_to_track` implementation doesn't really need any API calls provided by the client.

Reply:
Done, thanks